pkg/controller/hibernation/hibernation_controller.go

```diff
@@ -325,6 +325,11 @@ func (r *hibernationReconciler) checkClusterResumed(cd *hivev1.ClusterDeployment
 		logger.Warning("No compatible actuator found to check machine status")
 		return reconcile.Result{}, nil
 	}
+	// Ensure all machines have been started. Should have been handled already but we've seen VMs left in stopped state.
+	if err := actuator.StartMachines(cd, r.Client, logger); err != nil {
+		logger.WithError(err).Error("error starting machines")
+		return reconcile.Result{}, err
+	}
```

Comment:
Why are we not executing this code after the check for `running` machines? If machines are not running issue start command again

Reply:
Hmm good point, this would avoid an additional API call if they did happen to all be running. I'll drop it down.

Reply:
Updated!